blocks/blog-home/blog-home.js

```diff
@@ -168,41 +186,49 @@ function refreshCards() {
     const clearAllFilters = document.querySelector('.clear-all-filters');
     clearAllFilters.innerText = 'Clear all';
     clearAllFilters.addEventListener('click', () => {
-      clearFilters();
+      clearFilters(mode);
       deselectAllCheckboxes();
       clearAllFilters.innerText = '';
     });
 
-    checkedList.forEach((val) => {
+    checkedList.forEach((item) => {
       const selectedValue = createTag('div', { class: 'selected-value' });
-      selectedValue.append(val);
+      selectedValue.append(item.value);
       selectedFiltersList.append(selectedValue);
       selectedFiltersList.classList.add('active');
       // Add another event listener for click events to remove this item and uncheck the checkbox
       selectedValue.addEventListener('click', () => {
-        uncheckCheckbox(val);
+        uncheckCheckbox(item.value, mode);
         selectedValue.innerText = '';
       });
     });
   } else {
-    clearFilters();
+    clearFilters(mode);
   }
 }
 
-async function addEventListeners(checkboxes) {
-  checkboxes.forEach((checkbox) => checkbox.addEventListener('change', refreshCards));
+async function addEventListeners(checkboxes, mode) {
+  checkboxes.forEach((checkbox) => checkbox.addEventListener('change', () => refreshCards(mode)));
 }
 
-async function createCategories(categoriesList) {
+async function createCategories(categoriesList, mode) {
   const categoriesElement = createTag('div', { class: 'categories' });
   const catLabel = createTag('span', { class: 'category-title' });
-  catLabel.append('Categories');
+  catLabel.append(mode !== MODE ? 'Solutions' : 'Categories');
   categoriesElement.append(catLabel);
-  categoriesList.forEach((row) => {
+  const urlBypass = {
+    '/blog/enterprise-imaging': '/thought-leadership/solutions/merge',
+    '/blog/healthcare-analytics': '/thought-leadership/solutions/health-insights',
+    '/blog/real-world-evidence': '/thought-leadership/solutions/marketscan',
+    '/blog/clinical-development': '/thought-leadership/solutions/zelta',
+    '/blog/clinical-decision-support': '/thought-leadership/solutions/micromedex',
+    '/blog/health-human-services': '/thought-leadership/solutions/curam',
+  };
```

Comment:
I don't think we should explicitly bypass or define the URLs for the "Solutions" filter listing. The rendering of the Solutions List and its URLs should be filtered by any items in the query index that are part of the solution template (eg. [Health Insight landing](https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/Doc.aspx?sourcedoc=%7B493422CE-0B7C-4CBD-9482-7727FF908DA7%7D&file=health-insights.docx&action=default&mobileredirect=true)). Similar to what it's returning in the `getBlogCategoryPages()` https://github.com/hlxsites/merative2/blob/main/scripts/scripts.js#L369. 

@amol-anand perhaps there's some thing you can add to this as well?

Reply:
I agree with your recommendation, Putra. Page names/urls are always subject to change.

Reply:
@proeung in the "query-index.json" there is no `e.template === 'solution'` or any key-value pair where we have "Solution" as value.
All the paths in the json are in relation to '/blogs'. As mentioned before only place where `Thought leadership` is topics.

There is no way I can figure out where I can get thought leadership paths, It would be great if you can help me with that.
Also as i mentioned to you before, in "query-index.xlsx" file in root does not have any mention of new product( For example parent/product is still called `Clinical Development`, where as the documents you created are having updated name i.e. `zelta`), that is why I have to hard code these values. Can you help me with that too.

Reply:
I think unless the query-index file is not updated with "Thought Leadership" paths, it will be a blocker for me.

CC: @keith-kaplan @Shalini-SB

Reply:
Using `e.template === 'solution'` is the right approach here.
There are two ways to ensure the template metadata for the solution pages have the value 'solution'
1. Update the metadata per page to add the template value in each page
2. Bulk metadata - update metadata.xlsx for the /solutions/ path to add template = solution

If template is not specifically added in the metadata table for the pages under /solution, then I would go with option 2 and add it in the metadata.xlsx so authors don't have to deal with it as such.

Reply:
It looks like you already have the right template defined in the page metadata at the page level (option 1 from above). so the reason you don't see anything in the index is because these pages have not been published yet. Once you publish the pages under /thought-leadership, you will see them in the index and then your block will work.

@sahmad-merative @proeung